crate/README.MD

```diff
@@ -0,0 +1,73 @@
+# CrateDB Helm Chart
+
+This directory contains a Kubernetes chart to deploy a [CrateDB](https://crate.io) cluster using a StatefulSet.
+
+## Prerequisites Details
+* Kubernetes 1.5+
+* PV support on the underlying infrastructure
+
+## StatefulSet Details
+* https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/
+
+## StatefulSet Caveats
+* https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#limitations
+
+## Todo
+* 
```

Comment:
let's add: follow up with https://github.com/helm/charts/pull/8432

Reply:
didn't recall about it... not yet approved... pf :( if not we could just remove this chart

Reply:
Yeap, a ci issue no-one could help me with. looked like a time-out thing also happening to other people. Sure, we can come back to that at some point and when merged remove this one from here.